File: B53953/Laboratorio/Lab2/Prueba-1/src/main/java/com/orbis/b53953/business/LibroBusiness.java
package com.orbis.b53953.business;

import java.sql.SQLException;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.orbis.b53953.data.LibroData;
import com.orbis.b53953.domain.Libro;

@Service
public class LibroBusiness {

	@Autowired
	private LibroData libroData;
	
	public Libro save(Libro libro) throws SQLException{
		return libroData.save(libro);
	}
}

File: B63685/Labs/LabUnoASPNET/OrbisB63685Model/Business/LibroBusiness.cs
using OrbisB63685Model.Data;
using OrbisB63685Model.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace OrbisB63685Model.Business
{
   public class LibroBusiness
    {
        LibroData libroData = new LibroData();

        public IEnumerable<Libro> GetByCodEditorial(String codPublicador)
        {
            return libroData.GetByCodEditorial(codPublicador);
        }

        }
    }
